### PR TITLE
docs: Add a documentation for performance metrics

### PR DESCRIPTION
**Pull request by likebreath**
Signed-off-by: Bo Chen <chen.bo@intel.com>

### CODE REVIEW

File: docs/performance_metrics.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+$ ./scripts/dev_cli.sh tests --metrics -- -- --report-file /tmp/metrics.json
+```
+
+To get a list of available performance tests:
+
+```
+$ ./scripts/dev_cli.sh tests --metrics -- -- --list-tests
+```
+
+
```
Comment (sboeuf) [inline docs/performance_metrics.md:25]: Remove extra blank line.

File: docs/performance_metrics.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+To generate metrics data for selected performance tests, e.g. boot time only:
```
Comment (sboeuf) [inline docs/performance_metrics.md:26]: Add a blank line here if you want to be consistent with the previous blocks.

File: docs/performance_metrics.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Performance Metrics
+
+Cloud Hypervisor provides a [performance metrics](https://github.com/cloud-hypervisor/cloud-hypervisor/tree/main/performance-metrics)
+target binary for users to generate metrics data in their own
```
Comment (sboeuf) [inline docs/performance_metrics.md:4]:
```suggestion
binary for users to generate metrics data from their own
```

**Review by sboeuf — APPROVED** [review]

**Comment by sboeuf** [comment]
@rbradford could you please have a look?

**Review by rbradford — APPROVED** [review]

**Pull request merged**